### DISCUSSION TITLE
Keep freezing on windows when closing

### DISCUSSION THREAD

**Discussion by zer09** [wezterm/wezterm#4180] [General]
I'm having issue whene closing wezterm, it gets froozen after I choose "Yes" on "Really kill this window" message.

I'm running on windows 11, I am using wezterm to run WSL
here some details

Source
WezTerm - Wez's Terminal Emulator

Summary
Stopped responding and was closed

Date
‎8/‎21/‎2023 17:52

Status
Report sent

Description
A problem caused this program to stop interacting with Windows.
Faulting Application Path:	C:\Program Files\WezTerm\wezterm-gui.exe

Problem signature
Problem Event Name:	AppHangXProcB1
Application Name:	wezterm-gui.exe
Application Version:	1.0.0.0
Application Timestamp:	64aebbde
Hang Signature:	7e77
Hang Type:	134250496
Waiting on Application Name:	OpenConsole.exe
Waiting on Application Version:	0.0.0.0
OS Version:	10.0.22621.2.0.0.256.161
Locale ID:	1033
Additional Hang Signature 1:	7e7778e48250e5e36f3a649a6d731a4a
Additional Hang Signature 2:	0e30
Additional Hang Signature 3:	0e3097b69ad3ea090590a016728d8d35
Additional Hang Signature 4:	7e77
Additional Hang Signature 5:	7e7778e48250e5e36f3a649a6d731a4a
Additional Hang Signature 6:	0e30
Additional Hang Signature 7:	0e3097b69ad3ea090590a016728d8d35

Extra information about the problem
Bucket ID:	16e041f926956cb4d92f477e4cb2ec50 (1814747782631844944)

**Comment by h3athen** [comment]
I keep having the same problem

**Comment by DawieMalmsteel** [comment]
I still having the same problem with you on windows 11
